README.md

```diff
@@ -1,4 +1,4 @@
-# lifted-threads [![Hackage version](https://img.shields.io/hackage/v/lifted-threads.svg?label=Hackage)](https://hackage.haskell.org/package/lifted-threads) [![Build Status](https://secure.travis-ci.org/scrive/lifted-threads.svg?branch=master)](http://travis-ci.org/scrive/lifted-threads)
+# lifted-threads [![Hackage version](https://img.shields.io/hackage/v/lifted-threads.svg?label=Hackage)](https://hackage.haskell.org/package/lifted-threads)
```

Comment:
Replace build status with the github action badge, like e.g. [here](https://raw.githubusercontent.com/haskell-effectful/effectful/refs/heads/master/README.md).